packages/wrangler/src/index.tsx

```diff
@@ -100,6 +100,20 @@ function getScriptName(
     : shortScriptName;
 }
 
+async function requireAuth(config: Config): Promise<string> {
```

Comment:
open to suggestions on a better name for this function

Reply:
This name works for me!